application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadCommand.java

```diff
@@ -204,6 +205,18 @@ private void closeThread(SlashCommandInteractionEvent event, ThreadChannel helpT
         event.replyEmbeds(embed).flatMap(any -> helpThread.getManager().setArchived(true)).queue();
     }
 
+    private void bookmarkThread(SlashCommandInteractionEvent event, ThreadChannel helpThread) {
+        event.getUser()
+            .openPrivateChannel()
+            .flatMap(channel -> channel.sendMessage(String.format("<#%s>", helpThread.getIdLong())))
```

Comment:
move the part that creates the message into a dedicated help method please.

Reply:
```suggestion
            .flatMap(channel -> channel.sendMessage(helpThread.getAsMention())))
```

Reply:
the code is easier to read if its separated into groups (methods). and easier to maintain.
the message also might get more complex (see my other suggestion). all in all, its just easier if thinks are separated clean.

Reply:
Even if the message gets a little "bigger"

Separating it into another method only makes sense if it's huge, or contains logic. And 3 lines is really small, nor does it contain any _real logic_.

I mean
"""
your bookmarksssszz! :-
""" + helpChannel.getAsMention();

I personally much rather keep it like this, as separating into methods won't actually help readability, and will only make it harder / more complex when trying to read this command.


I agree with you, but not to this extent, it's I don't see the benefit at this level. 

So I'll leave it up to Taz to decide whenever they want to split it up or not.


Regarding "it's just easier ..."

Well, I very much doubt that, beginners tend to not use many methods at all, the more methods, the more complicated the code is. At least, that's what my experience says with beginners.



All in all, I don't disagree with splitting things up, I only believe that this method is too small to be eligible for that.